### PR TITLE
Node re-formatting

### PR DESCRIPTION
**Pull request by jney**

### CODE REVIEW

File: api/api/models/couchbase.js [hunk 1]
```diff
@@ -106,320 +114,282 @@ function verifyWorkspace (workspace) {
...
-      let payload = data
-      let request = { }
-      if (payload[key].request) {
-        request = {
-          action: payload[key].request.action,
-          ref: (payload[key].request.ref ? payload[key].request.ref : (payload[key].ref ? payload[key].ref : 'unknown')),
-          event: payload[key].request.event
+      return callback(err, null)
+    }
+    if (!data && !data[key]) {
```
Comment (gregoryguillou) [inline api/api/models/couchbase.js]: This should be `(!data || !data[key])` in that case, no?

Comment (jney) [reply api/api/models/couchbase.js]: fixing that

File: api/api/models/docker.js [hunk 2]
```diff
@@ -11,11 +11,9 @@ let startoptions = {}
...
 function version (callback) {
   const stdout = new EchoStream('version')
-  docker.run('deck-terraform', ['-v'], stdout, createoptions, startoptions, function (err, data, container) {
-    if (!err) {
-      callback(null, data)
-    } else {
-      callback(err, data)
+  docker.run('deck-terraform', ['-v'], stdout, createoptions, startoptions, (err, data) => {
+    if (err) {
+      return callback(err, data)
```
[the remark this answers is not in the document]
Comment (jney) [reply api/api/models/docker.js]: it depends on your workflow, but i think we should always send a callback

[the remark this answers is not in the document]
Comment (gregoryguillou) [reply api/api/models/docker.js]: Yes, that's my point; there is no callback if `(!err)`

**Pull request merged**